SpatialGDK/Extras/spot.version

```diff
@@ -1 +1 @@
-20191029.144741.87a7d78768
+20200120.115350.8d6b779c82
```

Comment:
this is the spot version, it is different from spatiald and they will not have the same versions.

Reply:
I believe this is what needs changing! https://github.com/spatialos/UnrealGDK/blob/91c7984c222e062e02322e8be5fb3d53bc7ed867/SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp#L23

Reply:
that is a very good point, changing now